pkg/store/proxy_test.go

```diff
@@ -55,7 +55,7 @@ func TestProxyStore_Info(t *testing.T) {
 	q := NewProxyStore(nil,
 		func() []Client { return nil },
 		component.Query,
-		nil, 0*time.Second,
+		nil, 0*time.Second, time.Minute,
```

Comment:
This will slowdown our typical test cases, could you move this out and protect by env variable?

https://github.com/improbable-eng/thanos/blob/master/pkg/store/proxy_test.go#L436-L439


I'm going to introduce optional CI jobs once we move to prow, so they won't be mandatory for merging.

Right now the plan is we run this as a period CI job. And doublecheck before doing releases.

Reply:
all good :+1:

This was due to CI speed as all the test cases in those 2 test tables would be slowed down by your test case.